crates/subspace-node/src/core_domain/cli.rs

```diff
@@ -74,6 +74,10 @@ pub struct CoreDomainCli {
     /// Additional args for core domain.
     #[clap(raw = true)]
     additional_args: Vec<String>,
+
+    /// Enable transaction sortition for the core domain.
+    #[arg(long)]
+    pub enable_tx_sortition: bool,
```

Comment:
Why this feature flag is unnecessary? It should just be the default behavior for every domain node IMO, and then the code can be simplified a lot.

Reply:
yeah the changes would be much simpler without the flag. My understanding was that new features should be flag protected, first enabled in devnet, then in prod.

But I am fine either way, will go with what folks suggest

Reply:
Removed cmd line flag